astropy/io/fits/hdu/table.py

```diff
@@ -999,14 +1009,27 @@ def dump(self, datafile=None, cdfile=None, hfile=None, clobber=False):
         plain text (ASCII) files.
         """
 
+        if isinstance(clobber, bool) and isinstance(overwrite, bool):
```

Comment:
Do you mean identical to the lines in `diff.py` and `file.py`?
I'm not sure whether it's worth creating a small function in e.g. `util.py`, or to have some code duplication. The three occurrences of this code reflect the three different output directions that exist in ``astropy.io.fits``.